air-quality-backend/tests/etl/forecast/forecast_adapter_test.py

```diff
@@ -2,17 +2,32 @@
 
 import pytest
 from cerberus import Validator
```

Comment:
Depending on whether you agree with some of the refactoring suggestions I think some of these would need to be updated/moved, and could patch out different items. 

For example if we move the 'file storage' parts out of process_variable into a separate file we can patch this out instead of Image.Save and (hopefully) the individual tests will end up simpler.

Reply:
adjusted the test accordingly and they seem to pass now again